integration-libs/opps/root/coupon-codes/opps-coupon-codes.module.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule, inject } from '@angular/core';
+import { LOCATION_INITIALIZED_MULTI } from '@spartacus/core';
+import { oppsCouponCodesInterceptors } from './http-interceptors';
+import { OppsCouponCodesService } from './opps-coupon-codes.service';
+
+export function saveCouponCodesFactory(): () => void {
+  return () => {
+    const service = inject(OppsCouponCodesService);
```

Comment:
Hmmm. In APP_INITIALIZERS you _have to_ inject stuff outside of the returned callback. But this is `LOCATION_INITIALIZED_MULTI` hook and perhaps here it's not forced.

But for consistency and readability, I recommend to do the same here -> let's swap above 2 lines.
Let's first let's inject all depenencies. Then let's return the callback.

Reply:
yes done